### PR TITLE
Add multi container deploys

### PR DESCRIPTION
**Pull request by sharangj**

### CODE REVIEW

File: bin/docker-deploy.sh [hunk 1]
```diff
@@ -57,12 +57,19 @@ return 0
...
- 
+
  if [ "$MULTIPLE_CONTAINER_PODS" == "true" ]; then
    KUBE_OPTS="$KUBE_OPTS --container=$app_name"
  fi
 
- kubectl rolling-update "$app_name" "--image=$repo:$tag" $KUBE_OPTS "--server=$KUBE_MASTER" "--namespace=$username" "--image-pull-policy=IfNotPresent"	
+ kubectl rolling-update "$app_name" "--image=$repo:$tag" $KUBE_OPTS "--server=$KUBE_MASTER" "--namespace=$username" "--image-pull-policy=IfNotPresent"
+
+ if [ "$MULTIPLE_CONTAINER_PODS" == "true" ]; then
```
Comment (gja) [inline bin/docker-deploy.sh]: can you avoid calling kubectl in this if check?

Basically,
if(multiple_container_pods == true)
  TO_DEPLOY= $DEPLOYABLE_CONTAINERS
else
  TO_DEPLOY="main"

File: bin/docker-deploy.sh [hunk 2]
```diff
@@ -57,12 +57,19 @@ return 0
...
+     for container in `echo $DEPLOYABLE_CONTAINERS | sed -n 1'p' | tr ',' '\n'`
```
Comment (gja) [inline bin/docker-deploy.sh]: yuk. Please see how to use arrays in bash https://stackoverflow.com/questions/10586153/split-string-into-an-array-in-bash

Comment (sharangj) [reply bin/docker-deploy.sh]: 👍

File: bin/docker-deploy.sh (hunk 1 above)
Comment (sharangj) [reply bin/docker-deploy.sh]: 👍

**Pull request merged**